src/graph/backend/elyzor/utils.hpp

```diff
@@ -78,4 +79,10 @@ std::vector<op_kind_t> get_supported_op_kinds();
 } // namespace graph
 } // namespace impl
 } // namespace dnnl
+
```

Comment:
declaring the operators outside of namespaces so they would be available everywhere